### ISSUE TITLE
Should all ESButtons have a pointer cursor

### ISSUE THREAD

**Issue by mansona** [ember-learn/ember-styleguide#362]
If you go to https://ember-styleguide.netlify.app/components/button and compare the cursor when hovering over the buttons in `Styling Links` section and any of the other buttons.

Essentially we have an inconsistency in if we show a pointer cursor. I don't know what the perfect solution to this is but a quick fix would be to make it so that all buttons have `cursor: pointer` on them so that at least they are consistent

does this have an a11y concern?

**Comment by prasannavijayan** [comment]
@mansona will check and raise PR.

**Comment by mansona** [comment]
Hey @prasannavijayan thanks for your contribution 🎉 

Sorry if I wasn't very clear, this issue wasn't actually ready to be implemented as there are some questions that need to be discussed and agreed upon before we make any change. For example, the question `does this have an a11y concern?` hasn't been answered and we're not going to make a change to all of our buttons without first answering that 👍 

If you're committed to making this change I would recommend starting a conversation in either #dev-ember-learning in the Discord or any of the A11Y channels in the Ember Discord and we can come up with a quick plan 👍 

Alternatively you could take an issue that is marked as `help-wanted` or doesn't have open questions if you're looking to contribute to the styleguide 😄 

Thanks again 🎉

**Comment by pichfl** [comment]
1. https://ux.stackexchange.com/questions/105024/why-dont-button-html-elements-have-a-css-cursor-pointer-by-default
2. https://adamsilver.io/blog/buttons-shouldnt-have-a-hand-cursor/

TLDR; I personally don't think actual buttons should have `cursor: pointer`. Having said that, I also think that the web is a moving target after all and I don't want to die on that hill.

**Comment by mansona** [comment]
After reading your material @pichfl I'm pretty convinced that this is a marginal change that we could probably justify either way. When there is a marginal change like this I tend to leave things as they are unless we have justification to make an active change. 

I'm going to close this for now and if we decide that we need to open it again at a later date then we can do that 👍
